cmd/helm/install.go

```diff
@@ -193,6 +194,7 @@ func newInstallCmd(c helm.Interface, out io.Writer) *cobra.Command {
 	f.StringVar(&inst.keyFile, "key-file", "", "identify HTTPS client using this SSL key file")
 	f.StringVar(&inst.caFile, "ca-file", "", "verify certificates of HTTPS-enabled servers using this CA bundle")
 	f.BoolVar(&inst.devel, "devel", false, "use development versions, too. Equivalent to version '>0.0.0-a'. If --version is set, this is ignored.")
+	f.BoolVar(&inst.depUp, "dep-up", false, "automatically execute helm dep up when charts are in requirements.yaml but not in charts.")
```

Comment:
Can we make this docstring (and the other references in this PR) a little clearer? Perhaps

> run `helm dependency update` before installing the chart

In documentation, we should always use the formal name rather than the short form. Additionally this shortens the verbiage... "when charts are in requirements.yaml but not in charts" is implied when using `helm dependency update`, so it's kinda redundant/confusing to the user.